app/src/main/java/com/ncnf/database/DatabaseServiceInterface.java

```diff
@@ -1,15 +1,30 @@
 package com.ncnf.database;
 
-import java.util.Map;
+import java.util.List;
 import java.util.concurrent.CompletableFuture;
 
 public interface DatabaseServiceInterface {
```

Comment:
It's great having an interface, but does more than one class implement it?

Reply:
Nope, this was to define the public interface of the Database